### DISCUSSION TITLE
Signal Integrity Channel Feature Extraction for Deep Learning (Project 198)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#29] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #198](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Signal%20Integrity%20Channel%20Feature%20Extraction%20for%20Deep%20Learning)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by chasedio10** [comment]
Hi, I am an electrical engineering student at the University of Maryland and interested in working and collaborating on this project. Feel free to comment on this post if you are interested in working together!

**Comment by nitish22066** [comment]
Hi, I'm a Physics and Electronics and Instrumentation student at Birla Institute of Technology and Science Goa, and I'm willing to contribute to this project and I'm eager to collaborate.

**Comment by anandr16910** [comment]
Hi, I am Anand from electrical communication engineering background from IISc Bangalore and would like to contribute to this project. Meanwhile I am doing some literature survey for this porject and would like to collaborate.
